nautobot/core/templates/inc/custom_fields_panel.html

```diff
@@ -17,6 +18,13 @@
                                 <a href="{{ value }}">{{ value|truncatechars:70 }}</a>
                             {% elif field.type == 'multi-select' and value %}
                                 {{ value|join:", " }}
+                            {% elif field.type == 'json' and value %}
+                                <p>
+                                    <button class="btn btn-xs btn-primary" type="button" data-toggle="collapse" data-target="#cf_json" aria-expanded="false" aria-controls="cf_json">
+                                        Show/Hide
+                                    </button>
+                                </p>
+                                <pre class="collapse" id="cf_json"><code>{{ value|render_json }}</code></pre>
```

Comment:
Oh huh. Won't this UI fail to work as expected if there are multiple JSON custom fields? They'll all have the same `id` and so the button will show/hide all of them at once, I think?

Probably the solution would be to embed `{{ field.name }}` in the `id` and `data-target` attributes here?